security/detect-bidi-characters triggers on i18n JSON

For example:
```
MediaWiki/extensions/VisualEditor/lib/ve/i18n/hyw.json
  45:48  warning  Detected potential trojan source attack with unicode bidi introduced in this code: '"‎Տեսնել"'  security/detect-bidi-characters
```

Given that this protects against "Trojan source" (https://github.com/eslint-community/eslint-plugin-security/blob/main/docs/rules/detect-bidi-characters.md), I don't see how this would apply to a JSON data file.

Furthermore as JSON doesn't support comments for inline disables, this can't be disabled per line.

--- a/json.json
+++ b/json.json
@@ -9,6 +9,7 @@
 		"json-es/use-valid-json": "error",
 		"indent": [ "error", "tab" ],
 		"strict": "off",
-		"max-len": "off"
+		"max-len": "off",
+		"security/detect-bidi-characters": "off"
 	}
 }

--- a/test/fixtures/json/valid.jsonc
+++ b/test/fixtures/json/valid.jsonc
@@ -1,6 +1,7 @@
 {
 	"max-len":
-																									[]// Off: max-len
+																									[], // Off: max-len
+	"bidi": "‎LTR"// Off: security/detect-bidi-characters
 }
 // Off: strict
 // Off: accessor-pairs (https://github.com/zeitport/eslint-plugin-json-es/issues/17)

